<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="edit_input">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>621</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_exchange">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>261</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>EXCHANGE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_md5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sha256">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA-256</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_sha521">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA - 512</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_md5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>621</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_sha256">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_sha512">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
